File: gherkinexecutor/Feature_Examples/ValueValidInternal.java
package gherkinexecutor.Feature_Examples;
import java.util.*;
class ValueValidInternal{
     String value;
     Boolean valid;
     String notes;
     
    public static String toDataTypeString() {
        return "ValueValidInternal {"
        +"String " 
        +"Boolean " 
        +"String " 
            + "} "; }  
    ValueValid toValueValid() {
        return new ValueValid(
        value
        ,String.valueOf(valid)
        ,notes
        ); }
    public ValueValidInternal(
        String value
        ,Boolean valid
        ,String notes
        )  {
        this.value = value;
        this.valid = valid;
        this.notes = notes;
        }
    @Override
    public boolean equals (Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        ValueValidInternal _ValueValidInternal = (ValueValidInternal) o;
         return 
                ( _ValueValidInternal.value.equals(this.value))
                 && ( _ValueValidInternal.valid.equals(this.valid))
                 && ( _ValueValidInternal.notes.equals(this.notes))
             ;  }
    @Override
    public String toString() {
        return "ValueValidInternal {"
        +"value = " + value + " "
        +"valid = " + valid + " "
        +"notes = " + notes + " "
            + "} " + "\n"; }  
    }

File: gherkinexecutor/Feature_Examples/ValueValidInternal.py
class ValueValidInternal:
    def __init__(self, value, valid, notes):
        self.value = value
        self.valid = valid
        self.notes = notes

    @staticmethod
    def to_data_type_string():
        return "ValueValidInternal { String Boolean String }"

    def to_value_valid(self):
        return ValueValid(self.value, str(self.valid), self.notes)

    def __eq__(self, other):
        if self is other:
            return True
        if other is None or not isinstance(other, ValueValidInternal):
            return False
        return self.value == other.value and self.valid == other.valid and self.notes == other.notes

    def __str__(self):
        return f"ValueValidInternal {{ value = {self.value} valid = {self.valid} notes = {self.notes} }}\n"

# Assuming ValueValid class is defined elsewhere
